src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -30,43 +24,31 @@ public LineService(LineRepository lineRepository, StationService stationService)
     }
 
     public LineResponse saveLine(LineRequest request) {
-        Station upStation = stationService.findById(request.getUpStationId());
-        Station downStation = stationService.findById(request.getDownStationId());
-        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
-        return LineResponse.of(persistLine, stations);
+        Station upStation = findStationById(request.getUpStationId());
+        Station downStation = findStationById(request.getDownStationId());
+
+        Line persistLine = lineRepository.save(Line.of(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
```

Comment:
LineRequest에 엔티티를 만드는 부분을 캡슐화해보는건 어떨까요? 🤔 

```suggestion
        Line persistLine = lineRepository.save(request.toLine(upStation, downStation));
```

Reply:
필요한 부분이었습니다!! 어떻게 수정해야 하는지 고민하고 있던 부분입니다. 감사합니다.